docs/general/networking/nginx.md

```diff
@@ -9,83 +9,145 @@ title: Nginx
 
 ## Nginx from a subdomain (jellyfin.example.org)
 
-:::caution
-
-HTTP is insecure. The following configuration is provided for ease of use only. If you are planning on exposing your server over the Internet you should setup HTTPS. [Let's Encrypt](https://letsencrypt.org/getting-started/) can provide free TLS certificates which can be installed easily via [certbot](https://certbot.eff.org/). Using only HTTP will expose passwords and API keys.
-
-:::
-
 :::tip
 
 The default X-Frame-Options header may cause issues with the webOS app, causing it to remain stuck at a black screen. If enabled, the default Content Security Policy may also cause issues.
 
 :::
```

Comment:
A caution is more appropriate for this one but realistically it can be removed altogether IMO, there is already mentions about it in the comments in the config